src/functions.php

```
@@ -399,58 +399,53 @@ function run(
-            $client = Deployer::get()->sshClient;
-            $output = $client->run($host, $command, $params);
-        }
-        return rtrim($output);
-    };
+    if (!empty($cwd)) {
+        $command = "cd $cwd && ($command)";
+    }
+
+    $runParams = new RunParams(
+        $command,
+        $timeout,
```

PHPStan (2.1.15)

Parameter #2 $cwd of class Deployer\Ssh\RunParams constructor expects string|null, int|null given.


(argument.type)